### PR TITLE
Merge gz-gui7 ➡️  main

### PR DESCRIPTION
**Pull request by azeey**
# ➡️  Forward port

Port `gz-gui7 ` ➡️  `main`

Branch comparison: https://github.com/gazebosim/gz-gui/compare/main...gz-gui7

**Note to maintainers**: Remember to **Merge** with commit (not squash-merge or rebase)

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -61,7 +61,7 @@ foreach(test ${gtest_targets})
 endforeach()
 
 if(TARGET UNIT_gz_TEST)
-  # Running `gz gazebo` on macOS has problems when run with /usr/bin/ruby
+  # Running `gz gui` on macOS has problems when run with /usr/bin/ruby
```
Comment (jennuine) [inline src/CMakeLists.txt:64]: Should this be `gz sim` since the original was `gz gazebo`?

**Review by jennuine — APPROVED** [review]

File: src/CMakeLists.txt (hunk 1 above)
Comment (azeey) [reply src/CMakeLists.txt:64]: Good catch. I think the change was made in https://github.com/gazebosim/gz-gui/pull/538. We'll have to fix it there as well.

Comment (azeey) [reply src/CMakeLists.txt]: Done in https://github.com/gazebosim/gz-gui/pull/545/commits/3efe68417897daaeff7617b9b700583d34a3fda8

**Review by ahcorde — APPROVED** [review]

**Comment by mjcarroll** [comment]
How unhappy are we expecting CI to be here?

**Comment by azeey** [comment]
The Linux test failures are known. The `INTEGRATION_camera_tracking` test should be fixed by https://github.com/gazebosim/gz-gui/pull/537 and `INTEGRATION_minimal_scene` should be fixed by https://github.com/gazebosim/gz-gui/pull/429. I don't know if we have a fix for `INTEGRATION_transport_scene_manager` yet.

The windows warning is not new  (https://build.osrfoundation.org/job/ign_gui-ci-win/148/msbuild/)

**Pull request merged**